.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and Push Backstage Docker Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to build and push
+        required: true
+        type: string
+  repository_dispatch:
+    types: [backstage_release]
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+
+      - name: Install Dependencies
+        run: yarn install --immutable
+
+      - name: Build Backend
+        run: |
+          yarn tsc:full
+          yarn build:all
```

Comment:
https://github.com/env0/env0-backstage-plugin/blob/1ee3008a7d2ac496ba63ed096b126fd27f22ed3e/packages/backend/Dockerfile#L7-L8

By the docker file you can do it only for the BE don't need to full
Your call